docs/book/src/roadmap.md

```diff
@@ -1,36 +1,21 @@
 # Cluster API Azure Roadmap
 
-This roadmap is a constant work in progress, subject to frequent revision. Dates are approximations. Features are listed in no particular order.
+The best place to see what's coming is in [the public milestones](https://github.com/kubernetes-sigs/cluster-api-provider-azure/milestones).  
+The next numbered milestone (i.e. **1.8**) is planned at the very beginning of the 2 month release cycle and most actively planned and discussed on the community call after the major release.
```

Comment:
nit: if we're using 1.8 as an example, then we should prefer the Latin abbreviation "e.g." (exempli gratia). "i.e." usually indicates "in other words..." (id est, literally it/that is, but I digress).

Ref: https://www.merriam-webster.com/words-at-play/ie-vs-eg-abbreviation-meaning-usage-difference

By the way I added this feedback because @mboersma was going to anyways (I win this time, Matt!)